.github/workflows/release.yml

```diff
@@ -5,6 +5,9 @@ on:
     branches:
       - master
 
+permissions:
+  contents: read
```

Comment:
I don't see the point of changing this workflow, which has a legitimate need for a token and runs only on merges to master. The other workflows make more sense to restrict.

Reply:
You don't think it's worth restricting the `update_website` job?  It installs a bunch of packages from PyPI.